app/routes/api/claim.js

```diff
@@ -338,8 +338,9 @@ export const claimHandlers = [
       validate: {
         payload: Joi.object({
           reference: Joi.string().valid().required(),
-          status: Joi.number().valid(5, 9, 10).required(),
-          user: Joi.string().required()
+          status: Joi.number().required(),
+          user: Joi.string().required(),
+          note: Joi.string().allow('')
```

Comment:
This one isn't consistent with application route above. I think they should probably both be optional